core/modules/admin_bar/admin_bar.inc

```diff
@@ -719,16 +719,22 @@ function admin_bar_theme_settings() {
     '#type' => 'checkboxes',
     '#title' => t('Enabled components'),
     '#options' => array(
-      'admin_bar.icon' => t('Icon menu'),
-      'admin_bar.menu' => t('Management menu'),
-      'admin_bar.search' => t('Search bar'),
-      'admin_bar.users' => t('User counts'),
-      'admin_bar.account' => t('Account links'),
+      'admin_bar.icon' => t('Home menu'),
+      'admin_bar.menu' => t('Administration menu'),
+      'admin_bar.search' => t('Menu search'),
+      'admin_bar.users' => t('Online'),
+      'admin_bar.account' => t('User account links'),
     ),
     '#default_value' => $config->get('components'),
     '#description' => t('These features will be enabled/visible in the administration bar. Untick the boxes to disable/hide them.'),
   );
 
+  $form['components']['admin_bar.icon']['#description'] = t('A specialized menu that links to the home page and provides links to perform various other admin tasks (e.g.: clear caches, run cron, etc.).');
+  $form['components']['admin_bar.menu']['#description'] = t('Displays the various administrative menu options in the admin bar.');
+  $form['components']['admin_bar.search']['#description'] = t('A drop-down version of the top-level links in the Administration Menu.');
```

Comment:
This is the wrong description.

Reply:
A search box for quickly finding specific items in the Administration Menu.
This is the correct one?

Reply:
Yep